images/build/Dockerfile

```diff
@@ -106,34 +105,34 @@ RUN apt-get update \
     && rm -rf /var/lib/apt/lists/*
 RUN mkdir -p /tmp/scripts
 COPY build/__nodeVersions.sh /tmp/scripts
-RUN chmod a+x /tmp/scripts/__nodeVersions.sh \
- && . /tmp/scripts/__nodeVersions.sh \
- && curl -sL https://git.io/n-install | bash -s -- -ny - \
- && ~/n/bin/n -d 4.4.7 \
- && ~/n/bin/n -d 4.5.0 \
- && ~/n/bin/n -d 4.8.0 \
- && ~/n/bin/n -d 6.2.2 \
- && ~/n/bin/n -d 6.6.0 \
- && ~/n/bin/n -d 6.9.3 \
- && ~/n/bin/n -d 6.10.3 \
- && ~/n/bin/n -d 6.11.0 \
- && ~/n/bin/n -d 8.0.0 \
- && ~/n/bin/n -d 8.1.4 \
- && ~/n/bin/n -d 8.2.1 \
- && ~/n/bin/n -d 8.8.1 \
- && ~/n/bin/n -d 8.9.4 \
- && ~/n/bin/n -d 8.11.2 \
- && ~/n/bin/n -d 8.12.0 \
- && ~/n/bin/n -d 8.15.1 \
- && ~/n/bin/n -d 9.4.0 \
- && ~/n/bin/n -d 10.1.0 \
- && ~/n/bin/n -d 10.10.0 \
- && ~/n/bin/n -d 10.14.2 \
- && ~/n/bin/n -d 10.16.3 \
- && ~/n/bin/n -d 12.9.1 \
- && ~/n/bin/n -d $NODE6_VERSION \
- && mv /usr/local/n/versions/node /opt/nodejs \
- && rm -rf /usr/local/n ~/n
+COPY images/build/__common.sh /tmp/scripts
+COPY images/build/installPlatform.sh /tmp/scripts
+RUN chmod +x /tmp/scripts/installPlatform.sh
+RUN cd /tmp/scripts \ 
+ && . ./__nodeVersions.sh \
+ && ./installPlatform.sh nodejs 4.4.7 \
```

Comment:
So we are installing the binaries that we put in storage. As of this PR I am doing this only in build image. Will send out a separate PR for updating the runtime images.